.circleci/config.yml

```diff
@@ -709,9 +709,15 @@ workflows:
 
       - test-contracts:
           name: test-contracts
+      - docker-build-and-push:
```

Comment:
we removed this because these containers aren't actually used anywhere. we made them nightly just so we have it but i think we can safely remove these on each commit

Reply:
If these are not used anywhere, is there a reason to build these at all?

Reply:
not really. to be safe thought we could keep nightly around for a few weeks before deleting just in case but i think these used to be built for the CI env we never used/no longer have.

Reply:
realistically i think we could delete cause we build our containers by hand for dev consumption and stage/prod we dont use containers

Reply:
Thanks for the clarification, I will move this back to nightly 👍